### ISSUE TITLE
CA2252 False positive on my own internal interface

### ISSUE THREAD

**Issue by saucecontrol** [dotnet/roslyn#55596]
**Version Used**: 

VS 16.11 Preview 4

**Steps to Reproduce**:

Seems to be happening with any interface defined in the same project.  For example:

```C#
namespace libtest;

public interface ISomething { }

public class Class1 : ISomething { }
```

**Expected Behavior**:

No warnings

**Actual Behavior**:

Warning CA2252

![snap_screen_20210812170159](https://user-images.githubusercontent.com/7929102/129284805-36dd4ac4-3fc7-4e62-a000-0f50a48b9846.png)

![snap_screen_20210812164854](https://user-images.githubusercontent.com/7929102/129284082-6a8d262d-a4b8-4063-9776-a373d592b827.png)

**Comment by RikkiGibson** [comment]
May be a dupe of dotnet/roslyn-analyzers#5366.

**Comment by saucecontrol** [comment]
Oh yes, I didn't notice the analyzers had their own repo.  I'll close this one.
